massa-models/src/denunciation_interest.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+use crate::endorsement::WrappedEndorsement;
+use crate::operation::WrappedOperation;
+use crate::block::WrappedHeader;
+
+/// What can be sent to Denunciation factory (in order to create Denunciation)
+#[derive(Debug)]
+pub enum DenunciationInterest {
```

Comment:
Maybe `DenunciationEvent` is a naming that match more the rest of the project

Reply:
I think DenunciationEvent is kinda misleading as here we regroup things that can later make a Denunciation.